infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -70,52 +83,13 @@ public SequencerServerCache(long cacheSize, CacheWriter<ConflictTxStream, Long>
      *
      * @param cacheSize cache size
      */
-    public SequencerServerCache(long cacheSize) {
-        this.conflictCache = Caffeine.newBuilder()
-                .maximumSize(cacheSize)
-                //Performing periodic maintenance using current thread (synchronously)
-                .executor(Runnable::run)
-                .writer(getDefaultCacheWriter())
-                .recordStats()
-                .build();
-    }
 
-    private CacheWriter<ConflictTxStream, Long> getDefaultCacheWriter() {
-        return new CacheWriter<ConflictTxStream, Long>() {
-            /**
-             * Don't do any additional actions during the write operation. Let caffeine write the record into the cache
-             * @param key a key
-             * @param value a value
-             */
-            @Override
-            public void write(@Nonnull ConflictTxStream key, @Nonnull Long value) {
-                //ignore
-            }
-
-            /**
-             * Eviction policy https://github.com/ben-manes/caffeine/wiki/Writer
-             * @param key  conflict stream
-             * @param globalAddress global address
-             * @param cause a removal cause
-             */
-            @Override
-            public void delete(@Nonnull ConflictTxStream key, Long globalAddress, @Nonnull RemovalCause cause) {
-                if (cause == RemovalCause.REPLACED) {
-                    String errMsg = String.format("Override error. Conflict key: %s, address: %s", key, globalAddress);
-                    throw new IllegalStateException(errMsg);
-                }
-
-                log.trace(
-                        "Updating maxConflictWildcard. Old = '{}', new ='{}' conflictParam = '{}'. Cause = '{}'",
-                        maxConflictWildcard, globalAddress, key, cause
-                );
-
-                if (globalAddress == null) {
-                    globalAddress = Address.NOT_FOUND;
-                }
-                maxConflictWildcard = Math.max(globalAddress, maxConflictWildcard);
-            }
-        };
+    public SequencerServerCache(int cacheSize, long maxConflictNewSequencer) {
```

Comment:
please remove part of javadoc (caffeine not used anymore)
![image](https://user-images.githubusercontent.com/314292/73117853-958fca80-3f00-11ea-8708-48ffeb566b7d.png)